src/Ansatz/Chain.jl

```diff
@@ -187,6 +187,33 @@ function canonize_site!(::Open, tn::Chain, site::Site; direction::Symbol, mode =
     return tn
 end
 
+function truncate!(qtn::Chain, bond; threshold::Union{Nothing,Real} = nothing, maxdim::Union{Nothing,Int} = nothing)
```

Comment:
```suggestion
function truncate!(qtn::Chain, bond::Union{Tuple{Site, Site}, Vector{Site}}; threshold::Union{Nothing,Real} = nothing, maxdim::Union{Nothing,Int} = nothing)
```
Maybe let's specify the type in `bond`

Reply:
It's better not to specify the type. Remember, the type should be just used for multiple dispatch.